q-pup_config/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
Should be 3.0.2 for FULL KINETIC BACKWARDS COMPATIBILITY

Reply:
>.< we target noetic and rep3 says noetic on focal is 3.16.3 :D

Reply:
that's saying the minumim the system must provide, so package must be compatible with at least that old. Older is fine, and increases compatability :)

We updated like all the packages from 2.8.3 to 3.0.2 last year because of some deprecation warnings, but 3.0.2 is the new "standard" version packages ought to depend use unless they have good reason to use newer features

Reply:
i mean tbf kinetic is no longer supported and i like shiny things :)))

Reply:
i updated to 3.0.2 ;)